### PR TITLE
Provide a more concise workflow in setup and "theoretical minimum" instructions

### PR DESCRIPTION
**Pull request by Ben-PH**
see #47

### CODE REVIEW

File: crate/guides/quickstart.md [hunk 1]
```diff
@@ -26,21 +26,25 @@ or `todomvc` examples.
 
 ## Setup
 
-This framework requires you to install [Rust](https://www.rust-lang.org/tools/install).
+The two fundamental things to setup:
 
-You'll need a recent version of Rust: `rustup update`
+ * [Rust](https://www.rust-lang.org/tools/install), the language itself.
+  * `rustup update` is also important
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:32]: There is probably an extra white-space before `*`.

And are you sure that users will know that they should run it in the  terminal? => I would change _"is also important"_ to something more valuable to users. (And _"is also important"_ statement isn't true if the user just installed Rust).

File: crate/guides/quickstart.md [hunk 2]
```diff
@@ -26,21 +26,25 @@ or `todomvc` examples.
...
+ * `cargo install cargo-make`, a broadly useful tool to for Rust [workflows](https://github.com/sagiegurari/cargo-make)
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:33]: There is missing dot at the end even though the sentence doesn't continue on the next line (like the case below).
We should make content as consistent as possible to make it more readable / scannable.

**Comment by MartinKavik** [comment]
- GitHub is clever enough to update PR when new commits are pushed:
  - Please close the previous PR https://github.com/seed-rs/seed-rs.org/pulls (if I'm not wrong, they are almost identical.)
  - Update the title of this PR to more descriptive name to make it easier to find in the future.
- Also please squash commits into one (with descriptive name, ideally respecting [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)). And force-push it. We rebase new PRs onto `master` so it will show in the project's commit history.

**Comment by Ben-PH** [comment]
Thanks for the feedback. How is that looking?

File: crate/guides/quickstart.md [hunk 3]
```diff
@@ -26,21 +26,25 @@ or `todomvc` examples.
 
 ## Setup
 
-This framework requires you to install [Rust](https://www.rust-lang.org/tools/install).
+The two fundamental things to setup:
 
-You'll need a recent version of Rust: `rustup update`
+ * [Rust](https://www.rust-lang.org/tools/install), the language itself.
+   * run `rustup update` from the cli: the project assumes a relatively new version of rust.
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:32]:
```suggestion
 * Run `rustup update` from the cli: the project assumes a relatively new version of Rust.
```

File: crate/guides/quickstart.md [hunk 4]
```diff
@@ -26,21 +26,25 @@ or `todomvc` examples.
...
 
-To start, clone [the quickstart repo](https://github.com/seed-rs/seed-quickstart):
-`git clone https://github.com/seed-rs/seed-quickstart.git`,
-run `cargo make build` in a terminal to build the app, and `cargo make serve` to start a dev server
-on [127.0.0.1:8000](http://127.0.0.1:8000). If you'd like the compiler to automatically check and recompile when you 
-make changes, run `cargo make watch` instead of `cargo make build`.
+The two main steps:
+
+ * [generate](https://github.com/seed-rs/seed-quickstart/generate) your own repo from this one as a template, then clone it.
+ * from the clones root-directory: `cargo make start` - builds the project and serves it to [localhost:8000](http://127.0.0.1:8000).
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:41]:
```suggestion
 * [Generate](https://github.com/seed-rs/seed-quickstart/generate) your own repo from this one as a template, then clone it.
 * From the clones root-directory: `cargo make start` - builds the project and serves it to [localhost:8000](http://127.0.0.1:8000).
```

**Comment by MartinKavik** [comment]
Please finish tasks in https://github.com/seed-rs/seed-rs.org/pull/48#issuecomment-625751675 and then request review.

**Pull request merged**